src/java/voldemort/common/nio/SelectorManagerWorker.java

```diff
@@ -122,12 +120,11 @@ else if(!selectionKey.isValid())
             reportException(e);
             close();
         } catch(IOException e) {
-            logger.info("Connection reset from " + socketChannel.socket() + " with message - "
-                        + e.getMessage());
+            logger.info("Connection reset from " + getDebugInfo() + " with message - " + e.getMessage());
```

Comment:
Agreed. I never thought about changing this error message, though I have been biten by this in the past. I will change it to IOException from